jekyll/_cci2/windows-images-support-policy.adoc

```diff
@@ -0,0 +1,105 @@
+
+- `edge`: tags are reserved for previews of new releases, which will initially point to this tag. The `edge` tags may include incremental updates to the current quarterly image release, which may change without notice, and is not recommended to be used for production CI workloads. `current` will be updated with these changes after period of stability.
+
+- `previous`: version of Windows Server will receive the previous version of `current`.
+
+We generally install the latest Major version of Visual Studio Community Edition that coincides with the Windows Server version in the image when it is built.
+
+We always aim to release a version of the supported windows images (e.g. currently this is `Windows server 2019` and `Windows server 2022`) every 3 months. We recommend using the `default` version and not pinning to a date version.
+
+== Tagging with the Windows Orb:
+
+The variant of Windows Server can be specified by choosing the name parameter in the Windows Orb. The most recent version of Windows Server will always be accessible via the `win/default` name, but previous versions require explicit definition, such as `win/server-2019`.
+
+Each quarterly (published every 3 months) image for the current tag Windows Server version is published to a date-based tag, such as 2023.10.1, which can be explicitly defined in the version parameter of the Orb.
+
+[#critical-cve-patches]
+== Critical CVE patches
+
+When critical CVEs are disclosed that affect the versions of the operating system or software stack in our Windows Images, we will investigate the impact that this has on our images being used within the CircleCI execution environment. If customers are impacted by these CVEs we will push a patch fix to the released image(s), this image will supersede the original image.
+
+[#bug-reports-issues-and-prs]
+== Bug reports, issues, and PRs
+
+You can file a link:https://support.circleci.com/hc/en-us/requests/new[support ticket] with CircleCI Support for any issues or bugs found with the Windows images. Our support team will be able to escalate issues internally to the correct engineering team and provide updates on the issue.
+
+[#image-lifespan-eol]
+== Image lifespan / EOL
+
+Once the next major version of Windows Server is released by Microsoft and generally available to customers, we will deprecate the oldest version of Windows Server that we support and schedule it to be removed.
+
+After the new version of Windows Server has been released, we will then move the current Windows Server version into an extended support phase and move to the tagging system described in the Tagging section of this document.
+
+Current Deprecation (Jan 2024):
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+| Version
+| Support
+
+| Windows Server 2019
+| Build quarterly images and retain Q4 (October) releases from previous two years
+
+| Windows Server 2022
+| Build quarterly images and Q4 (October) releases retained
```

Comment:
We wanted specific wording for this to show we will only keep Q4 image each year: `Quarterly images built and only Q4 image retained each year`